assets/app/features/upgrade/upgrade.constants.js

```diff
@@ -12,6 +12,7 @@
             'ha': ['clusters_healthy'],
             'ceph': ['ceph_healthy']
         })
+        .constant('UPGRADE_LAST_STATE_KEY', 'soc.upgrade.lastUIState')
```

Comment:
It's a key under which last upgrade state will be stored in localStorage. Those are scoped to the protocol+domain context so we need to make sure it doesn't cause conflicts with other parts of application.